src/mongo/MongoScrapbook.ts

```diff
@@ -405,7 +411,7 @@ class FindMongoCommandsVisitor extends MongoVisitor<MongoCommand[]> {
 		}
 	}
 
-	private addErrorToCommand(error: IParsedError, ctx: mongoParser.ArgumentContext | mongoParser.PropertyValueContext): void {
+	private addErrorToCommand(error: { message: string }, ctx: mongoParser.ArgumentContext | mongoParser.PropertyValueContext): void {
```

Comment:
Ah, I see.  Silly to create an error just to pull out them message, but don't change it for the patch.